album_cover_generator.py

```diff
@@ -0,0 +1,22 @@
+import io
+import PIL
+from PIL import ImageFilter, Image
+
+def generate_final_cover(cover, coverFilename):
```

Comment:
The naming of "coverFilename" is different from the others.

Reply:
Changed!